.github/workflows/railway-deploy.yml

```
@@ -0,0 +1,188 @@
+            exit 1
+          fi
+          echo "✅ Build artifacts verified"
+
+      - name: 🧪 Test server configuration
+        run: |
+          cd server
+          timeout 5 node index.js > /dev/null 2>&1 || true
+          echo "✅ Server configuration valid"
+
+      - name: 📊 Build size report
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:1:34: Double quote to prevent globbing and word splitting

(shellcheck)